index.js

```diff
@@ -184,7 +184,7 @@ SonosAccessory.prototype.search = function() {
 
         this.log("Found a playable device at %s for room '%s'", host, roomName);
         this.device = device;
-        search.socket.close(); // we don't need to continue searching.
+        search.destroy(); // we don't need to continue searching.
```

Comment:
The current version that works with Sonos 9.1 doesn't actually work because while it find the accessories properly, it doesn't stop the search properly which causes a crash when the search times out.